### ISSUE TITLE
There is a CJK font family: Google Noto CJK.

### ISSUE THREAD

**Issue by diguage** [asciidoctor/asciidoctor-pdf#1048]
[Noto CJK – Google Noto Fonts](https://www.google.com/get/noto/help/cjk/), All Noto fonts are published under the SIL Open Font License, Version 1.1.

[chloerei/asciidoctor-pdf-cjk-kai_gen_gothic](https://github.com/chloerei/asciidoctor-pdf-cjk-kai_gen_gothic) is not updated for four years, and KaiGen Gothic fonts were deleted on GitHub.

**So maybe we can use Google Noto Fonts for CJK to resolve the problem, maybe design a theme based on Google Noto Fonts and so on.**

**Comment by mojavelinux** [comment]
I agree. Feel free to either fork that project or create a new one that provides a CJK theme. Then we can document it in the README.

Given the size of the CJK fonts, I think it warrants a separate package that just provides the theme.

**Comment by mojavelinux** [comment]
I have documented how to use the Noto CJK font in the following new guide. https://github.com/asciidoctor/asciidoctor-pdf/blob/main/docs/modules/theme/pages/cjk.adoc This guide will be included in https://docs.asciidoctor.org when Asciidoctor PDF 2.0.0 is released.

Unfortunately, the Noto project only provides a single TTF for each family (Sans and Serif). If they were to make TTFs available for all weights, they could be much more easily used in Asciidoctor PDF. Right now, the options are limited. I tried the OTF fonts and ttfunk still isn't able to use them. So only the TTF formats can be used.
